api/api/specs/routers.yaml

```diff
@@ -13,7 +13,10 @@ info:
 .timeout: &timeout
   type: "string"
   pattern: '^[0-9]+(ms|s|m|h)$'
-  
+
+.deployFlag: &deployFlag
+  type: boolean
```

Comment:
This doesn't make sense. Such YAML anchors are only added for the re-usability (meaning, for schemas, that are used more than once). This can be just defined on the operation itself